src/translator/quality_estimator.cpp

```diff
@@ -160,7 +160,7 @@ Response::SentenceQualityScore LogisticRegressorQualityEstimator::computeSentenc
   const float sentenceScore =
       std::accumulate(std::begin(wordScores), std::end(wordScores), float(0.0)) / wordScores.size();
 
-  return {wordScores, subwordToWords(wordIndices, target, sentenceIdx), sentenceScore};
+  return {wordScores, wordIndices, sentenceScore};
```

Comment:
> #357 fixes this by making quality estimates code store references to the text in the same way as alignment etc. does: metadata always attached at the subword level.

@jelmervdl I assume you're talking about this. I agree instances of the above are an improvement.